### PR TITLE
Add KafkaExceptionMapper

### PR DESCRIPTION
**Pull request by omkreddy**
- Add a common Kafka Exception Mapper which can be be used all rest-projects

### CODE REVIEW

File: core/src/main/java/io/confluent/rest/exceptions/KafkaExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
```
Comment (rajinisivaram) [inline core/src/main/java/io/confluent/rest/exceptions/KafkaExceptionMapper.java]: Is this the right license for files in `rest-utils`? Other classes seem to have Apache license.

**Comment by rajinisivaram** [comment]
The PR build failure looks related to the changes.

**Comment by omkreddy** [comment]
> The PR build failure looks related to the changes.
test failure is not related to this change. looks like transient test failure,

**Comment by omkreddy** [comment]
Merging to master.

**Pull request merged**